modules/core/shared/src/main/scala/lucuma/core/util/Display.scala

```diff
@@ -17,23 +17,23 @@ object Display {
 
   /** Create an instance of Display using the provided functions. 
    * 
-   * @param toShort function that maps A to the shortName
-   * @param toLong function that maps A to the longName
+   * @param toShortName function that maps A to the shortName
+   * @param toLongName function that maps A to the longName
   */
-  def from[A](toShort: A => String, toLong: A => String): Display[A] = 
+  def from[A](toShortName: A => String, toLongName: A => String): Display[A] = 
```

Comment:
Contravariant instance constructors like this are conventionally called `by` so I suggest `by` and `byShortName` for these.

Reply:
Will do. Wasn't certain about the naming, anyway.

Reply:
@tpolecat What does it make it `Contravariant`?

Reply:
@tpolecat Do you want just the constructors themselves renamed, or the parameters as well?